src/metabase/driver.clj

```diff
@@ -32,7 +32,9 @@
 
 (def ^:const connection-error-messages
   "Generic error messages that drivers should return in their implementation of `humanize-connection-error-message`."
-  {:cannot-connect-check-host-and-port "Hmm, we couldn't connect to the database. Make sure your host and port settings are correct."
+  {:cannot-connect-check-host-and-port "Hmm, we couldn't connect to the database. Make sure your host and port settings are "
```

Comment:
Why are we cutting off the sentence? Make sure your host and port settings are what?